app/concerns/permission_checking.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# These methods are used for checking Permissions (aka ACLs) to determine if
+# a user has access to System or Provider Identity Permissions
+# these will be used by the controllers as well as Pundit
+module PermissionChecking
+  extend ActiveSupport::Concern
+
+  def is_non_nasa_draft_user?(user:, token:)
```

Comment:
I don't see an "else" if all the conditions above are false

Reply:
I guess you don't. have to explicitly return false in rails

Reply:
No, I think it works without, but you're right it would be better to have it be there explicitly.

Reply:
Actually, I don't think there can be an else case here, it is just running through all the environments. The true or false is coming from the different `user_has_provider_permission_to` calls for the environments.